pkg/controller/account/account_controller.go

```diff
@@ -582,6 +581,7 @@ func checkAWSAccountsLimitReached(r *ReconcileAccount, reqLogger logr.Logger, cu
 	} else {
 		if limit, ok := instance.Data["account-limit"]; ok {
 			if i, err := strconv.Atoi(limit); err == nil {
+				reqLogger.Info(fmt.Sprintf("Amount of Current Accounts: %d -- Account Limit: %d", currentAccounts, i))
```

Comment:
Super anal non-code femtonit.
```suggestion
				reqLogger.Info(fmt.Sprintf("Number of Current Accounts: %d -- Account Limit: %d", currentAccounts, i))
```